framework/utils/randomUtils.py

```diff
@@ -192,12 +193,28 @@ def randomIntegers(low, high, caller, engine=None):
     rawNum = low + random(engine=engine)*intRange
     rawInt = int(round(rawNum))
```

Comment:
Is round correct here, or would floor or ceil be better? I'm worried that it is biasing it away from some values.

Reply:
Hi Josh. I think it should be fine here since the random returns a float between 0 1